<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contcalcdlg</class>
 <widget class="QDialog" name="contcalcdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Continuum calculations</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="entirespec">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use the whole spectrum for evaluation rather than the
specified &quot;include&quot; ranges.

Note that the excluded ranges always apply.</string>
   </property>
   <property name="text">
    <string>Use entire spectrum for evaluation</string>
   </property>
  </widget>
  <widget class="QComboBox" name="inclrange2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Possibly put a range to include here (if not using whole spectrum)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="inclrange1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Possibly put a range to include here (if not using whole spectrum)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="exclrange1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Possibly put a range to exclude here</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Include</string>
   </property>
  </widget>
  <widget class="QComboBox" name="inclrange3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Possibly put a range to include here (if not using whole spectrum)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="exclrange2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Possibly put a range to exclude here</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Exclude</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="lowerlim">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Gives the lower limit of acceptable standard deviations to
exclude from the intensity after each iteration.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>3.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Lower limit
(std devs)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="upperlim">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Gives the upper limit of acceptable standard deviations to
exclude from the intensity after each iteration.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Upper limit
(std devs)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="degree">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>55</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set degree of poloynomial to fit</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Degree</string>
   </property>
   <property name="buddy">
    <cstring>degree</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="iterations">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>55</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set number of iterations to run before displaying</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Iterations</string>
   </property>
   <property name="buddy">
    <cstring>degree</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Ref wavelength</string>
   </property>
   <property name="buddy">
    <cstring>refwavel</cstring>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="refwavel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>This is the reference wavelength to be subtracted from the wavelength
values rather than basing the whole polynomial in terms of 6000+ for X.</string>
   </property>
   <property name="minimum">
    <double>6000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>6997.000000000000000</double>
   </property>
   <property name="value">
    <double>6561.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="restart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Restart the iterations from the beginning</string>
   </property>
   <property name="text">
    <string>Restart from beginning</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>restart</tabstop>
  <tabstop>degree</tabstop>
  <tabstop>iterations</tabstop>
  <tabstop>refwavel</tabstop>
  <tabstop>entirespec</tabstop>
  <tabstop>inclrange1</tabstop>
  <tabstop>inclrange2</tabstop>
  <tabstop>inclrange3</tabstop>
  <tabstop>exclrange1</tabstop>
  <tabstop>exclrange2</tabstop>
  <tabstop>upperlim</tabstop>
  <tabstop>lowerlim</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>contcalcdlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>contcalcdlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>696</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
